server/cmd/db-migrations/main.go

```diff
@@ -48,10 +48,16 @@ func main() {
 		return
 	}
 
+	// get db name
+	dbName := os.Getenv("REEARTH_CMS_DB_NAME")
```

Comment:
There is already an existing flag named `REEARTH_CMS_DB_CMS`, I think we can use it